buyer_data/models.py

```
@@ -71,3 +72,16 @@ def __str__(self):
         """
         return f"{self.buyer.user.email} bookmarked {self.estate.estate_name}"
+
+
+@receiver(models.signals.post_save, sender=WishlistItem)
+def increment_bookmarks(sender, instance, created, **kwargs):
+    if created:
+        with transaction.atomic():
+            estate_metrics, _ = EstateMetrics.objects.get_or_create(estate=instance.estate)
+            EstateMetrics.objects.select_for_update().filter(pk=estate_metrics.pk).update(bookmarks=models.F('bookmarks') + 1)
+
+@receiver(models.signals.post_delete, sender=WishlistItem)
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)